c/tskit/tables.c

```diff
@@ -267,7 +267,7 @@ tsk_individual_table_expand_location(
         = TSK_MAX(additional_length, self->max_location_length_increment);
```

Comment:
unrelated, but this looks wrong? I would think the increment should be `MAX(length, min_length)`? Or is `max_location_length_increment` poorly named?

Reply:
Oh, never mind - I see it's (max_location_length) increment, not max (location_length_increment).